debian/rules

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/make -f
+
+VERSION=0.1
+DEB_DIR=debian/streamgraph$(VERSION)
+
+%:
+	dh $@
+
+override_dh_auto_clean:
+
+override_dh_auto_build:
+	dh_auto_build
+
+override_dh_auto_install:
+	mkdir -p $(DEB_DIR)/usr/bin
+	install -m 755 obj-x86_64-linux-gnu/bin/streamgraph $(DEB_DIR)/usr/bin/streamgraph$(VERSION)
```

Comment:
Does the `obj-x86_64-linux-gnu` originate from `cmake` or does `debhelper` pass it into `cmake`?

In any case I don't think it's a big deal.